modules/customerprint.php

```diff
@@ -133,8 +133,8 @@
 		}
 
 		$SMARTY->assign('contactlist', $DB->GetAllByKey('SELECT customerid, MIN(contact) AS phone
-				FROM customercontacts WHERE contact <> \'\' AND type < ? GROUP BY customerid',
-				'customerid', array(CONTACT_EMAIL)));
+				FROM customercontacts WHERE contact <> \'\' AND ((type & ? = ? OR type & ? = ? OR type & ? = ?) AND type & ? <> ?) GROUP BY customerid',
+				'customerid', array(CONTACT_MOBILE, CONTACT_MOBILE, CONTACT_FAX, CONTACT_FAX, CONTACT_LANDLINE, CONTACT_LANDLINE, CONTACT_DISABLED, CONTACT_DISABLED)));
```

Comment:
Wszystkie te warunki można uprościć - napisałem we wcześniejszych notatkach w jaki sposób.